src/Api/Customer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Api;
+
+use Billogram\Api;
+use Billogram\Exception\Domain\ValidationException;
+use Billogram\Exception\InvalidArgumentException;
+use Billogram\Model\Customer\Customer as Model;
+use Billogram\Model\Customer\Customers;
+use Billogram\Api\HttpApi;
+
+class Customer extends HttpApi
+{
+    /**
+     * @param array $param
+     * @return string|array
+     * @link
+     */
+    public function search(array $param = ['page'=>1, 'page_size'=> 100 ])
```

Comment:
What if I do `$customer->search(['page_size'=>20]);`?
Now I would get a 400  since "page" is missing. 

Maybe we should add the default values in some other way...